src/physlr-makebf.cc

```diff
@@ -0,0 +1,161 @@
+#include <fstream>
+#include <getopt.h>
+#include <iomanip>
+#include <iostream>
+#include <sstream>
+#include <string>
+#include <vector>
+
+#if _OPENMP
+#include <omp.h>
+#endif
+
+#include "btl_bloomfilter/BloomFilter.hpp"
+#include "btl_bloomfilter/vendor/ntHashIterator.hpp"
+
+static void
+printErrorMsg(const std::string& progname, const std::string& msg)
+{
+	std::cerr << progname << ": " << msg << "\nTry 'physlr-makebf --help' for more information.\n";
+}
+
+static void
+printUsage(const std::string& progname)
+{
+	std::cout << "Usage:  " << progname
+	          << "  -k K [-v] [-o FILE] FILE...\n\n"
+	             "  -k K       use K as k-mer size\n"
+	             "  -b B       Bloom Filter size in Bytes\n"
+	             "  -v         enable verbose output\n"
+	             "  -o FILE    write Bloom filter to FILE [required]\n"
+	             "  -t N       use N number of threads\n"
+	             "  --help     display this help and exit\n"
+	             "  FILE       space separated list of ntHits output files\n";
```

Comment:
What file format for the ntHits output?

Reply:
```
AATGGAATGGAATGGAACGAAATGTAATGTAATGGAATGG        143
AAACCAAACAGTAAACCAAATCCAATGAAAATAGAAGTCA        168
AATGGAATAGAAAGGAATGAACCCGAAGGGTATGGAATGG        147
ACAGCACAATTAACATAACACAACACAGCATAACACAACA        336
ACGTGAACATATACCCGTTTCGAATGAAGGCCAGCCAGTG        224
AAGGTGTGAACCCGGGAGGCGGAGCTTGCAGTGAGCCGAG        132
```
Example

Reply:
You can do ntHits output in various formats thought, right? I was thining we might want to specify we want the TSV output.